src/test/resources/UserRegistration.feature

```diff
@@ -0,0 +1,47 @@
+Feature: User Registration
+  As a user (/admin)
+  I want to be able to register (users) with username and password
+
```

Comment:
Dont forget the password already taken case :)

Reply:
XDDD

Reply:
Scenario: Failed registration with used username, arbitrary password and password confirmation.
    When user requests registration with username "user", password "pig-system" and password confirmation "pig-system" with accessToken "accessToken"
    Then response status code is 409
    And response contains key "message" and value "Password already exists for user 'kangaroo'. If you are 'kangaroo' please use the log-in interface 'myNAS.localhost:3030/login'"
    And response contains key "status" and value "conflict"